tests/kernel/wave/wave_gemm_test.py

```diff
@@ -51,10 +52,43 @@ def get_test_shapes(test_name: str) -> list[tuple[int]]:
     return default_test_shapes[test_name]
 
 
+def get_load_elems_per_thread(mfma_variant: MMAType) -> int:
```

Comment:
Seems useful in life, maybe we cna move this into utils?

Reply:
sure, will move into utils.

Reply:
done!